roofit/roofitcore/src/RooAbsReal.cxx

```diff
@@ -2660,9 +2673,9 @@ double RooAbsReal::getPropagatedError(const RooFitResult &fr, const RooArgSet &n
   TMatrixDSym C(paramList.getSize()) ;
   vector<double> errVec(paramList.getSize()) ;
```

Comment:
here better using std::vector

Reply:
Okay! I have done the same for all `std::` classes in the file